app/app.py

```diff
@@ -1,39 +1,74 @@
 import os
 from statistics import mean
 
+from flask import Flask, render_template, flash
 from flask_migrate import Migrate
-from flask import Flask
 
 from flask_sqlalchemy import SQLAlchemy
 
+from werkzeug.utils import redirect
+
+from app.forms import NewRatingForm
+
 app = Flask(__name__)
 app.config.from_object(os.environ["APP_SETTINGS"])
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 db = SQLAlchemy(app)
 migrate = Migrate(app, db)
 
-from app.models import Restaurant
+from app.models import Restaurant, Rating
 
 
-@app.route("/")
+@app.route('/')
+@app.route("/index")
```

Comment:
dlaczego dwa routy tutaj? Chcemy żeby to było dostępne z dwóch podstron?

Reply:
taki był pomyśł, ale nie przemyślałem tego, myślałem, że na początku był tylko index, więc ja dodałem "/", ale widzę, że wczesniej był tylko "/", więc zostawię tak jak było